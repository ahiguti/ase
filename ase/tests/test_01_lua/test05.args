arg1 arg2 漢字x foo
