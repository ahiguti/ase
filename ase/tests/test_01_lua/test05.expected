nargs=4
arg 0 arg1
arg 1 arg2
arg 2 漢字x
arg 3 foo
